cmd/all-in-one/main.go

```diff
@@ -194,7 +195,7 @@ by default uses only in-memory database.`,
 			agent := startAgent(cp, aOpts, logger, metricsFactory)
 
 			// query
-			querySrv := startQuery(
+			querySrv, tracer := startQuery(
```

Comment:
why do you create a tracer in startQuery when there is initTracer function called in L107?

Reply:
@yurishkuro Can we rmv the initTracer() in favor of jtracer.New() ? Sorry if I misunderstood here.

Reply:
I'd say we should replace it, not remove. It makes no sense why a method `startQuery` would return a tracer object, it breaks all single purpose principles.